certbot/auth_handler.py

```diff
@@ -306,43 +197,18 @@ def _get_chall_pref(self, domain):
         chall_prefs.extend(plugin_pref)
         return chall_prefs
 
-    def _cleanup_challenges(self, aauthzrs, achalls=None):
+    def _cleanup_challenges(self, achalls):
         """Cleanup challenges.
 
-        :param aauthzrs: authorizations and their selected annotated
+        :param authzrs: authorizations and their selected annotated
             challenges
-        :type aauthzrs: `list` of `AnnotatedAuthzr`
+        :type authzrs: `list` of `AnnotatedAuthzr`
```

Comment:
This parameter is no longer passed to this method.

Reply:
Corrected.